index.bs

```diff
@@ -1708,6 +1773,20 @@ Attributes</h4>
 
 	</ins>
 </div>
+
+	: <dfn>sinkId</dfn>
+	::
+		This attribute contains the ID of the current audio output device, or the
+		empty string for the user-agent default device. The attribute MUST return
+		the value of {{AudioContext/[[sink ID]]}} internal slot upon getting.
```

Comment:
I think that it's better to only say that this returns the value of `[[sink ID]]` and not have too much prose, to not have to repeat what the values can be (e.g., this can also now return `"none"`).

Reply:
Yes, now I can make this simplification.